dashboard/app/models/user.rb

```diff
@@ -500,7 +501,8 @@ def district_teachers(cohort = nil)
   validates :name, length: {within: 1..70}, allow_blank: true
   validates :name, no_utf8mb4: true
 
-  defer_age = proc {|user| user.provider == 'google_oauth2' || user.provider == 'clever' || user.provider == User::PROVIDER_SPONSORED}
+  defer_age = proc {|user| %w(google_oauth2 clever powerschool).include?(user.provider) || user.provider == User::PROVIDER_SPONSORED}
```

Comment:
This allows a Powerschool signin to immediately complete, and pop up an age dialog later.